README.md

```diff
@@ -692,6 +813,7 @@ Java Versions
 -------------
 
 Java 7 or above is required for using this client.
+> Note: GAE standard for Java 7 is not supported for gRPC-based clients.
```

Comment:
Remove this note since https://github.com/GoogleCloudPlatform/google-cloud-java/pull/1982 already addressed this.